### PR TITLE
ADD: change graffiti

### PR DESCRIPTION
**Pull request by NeoPlays**
Changing graffitis for all keys for Lighthouse, Prysm and Teku Nodes is now possible. 
On Nimbus nodes the only way to change the graffiti while the service is running, is via their rest api.
Doing this via ansible and curl would slow things down a lot, so i left it open for now

### CODE REVIEW

**Comment by stefa2k** [comment]
This update means we need to change the existing services, right?

**Comment by NeoPlays** [comment]
> This update means we need to change the existing services, right?

yes for the graffiti file attribute

**Comment by stefa2k** [comment]
Okay, that's still to do then, same with #737

File: launcher/src/backend/ValidatorAccountManager.js [hunk 1]
```diff
@@ -228,4 +228,42 @@ export class ValidatorAccountManager {
         return URL
     }
 
+    async setGraffitis(graffiti){
```
Comment (stefa2k) [inline launcher/src/backend/ValidatorAccountManager.js:231]: it's possible to run mulitple validator services

**Review by stefa2k — CHANGES_REQUESTED** [review]

File: launcher/src/backend/ValidatorAccountManager.js (hunk 1 above)
Comment (NeoPlays) [reply launcher/src/backend/ValidatorAccountManager.js:231]: done 👍🏼

**Review by stefa2k — CHANGES_REQUESTED** [review]
There should be a verify of the validator services when started creating a `graffiti.yaml` in the molecule tests.

**Review by stefa2k — APPROVED** [review]
Nice work! 🚀

**Pull request merged**